build.backend.sh

```
@@ -0,0 +1,36 @@
+# force isatty-->true, so that $0 |& head -50 has colors ...
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)